tests/helpers/test_get_cinder_major_version.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+import pytest_rpc.helpers
+import testinfra.backend.base
+import testinfra.host
+
+"""Test cases for the 'get_cinder_major_version' helper function."""
+
+
+def test_valid_version(mocker):
+    """Verify ping_ip_from_utility_container returns True when the ping is
```

Comment:
```suggestion
    """Verify valid version ...
```

Reply:
🤦‍♂️ sorry for the copy/pasta errors @phongdly . Thank you so much for spotting them. I have patched this in my updated commit.